net/src/client/http/mod.rs

```diff
@@ -199,7 +199,14 @@ impl TryFrom<WitnetHttpResponse> for surf::http::Response {
         // Create a surf response and set all the relevant parts
         // Version and headers are currently not used, but they are included here for completion and
         // future readiness.
-        let mut res = surf::http::Response::new(status);
+        let code = status.status.as_u16();
+        let mut res =
+            std::panic::catch_unwind(|| surf::http::Response::new(status)).map_err(|_| {
```

Comment:
Could we add a comment explaining that we need to use catch_unwind  because the surf library panics instead of returning a `Result`?